batch-submitter/drivers/proposer/driver.go

```diff
@@ -33,7 +33,6 @@ import (
 	rollupTypes "github.com/mantlenetworkio/mantle/fraud-proof/rollup/types"
 	l2types "github.com/mantlenetworkio/mantle/l2geth/core/types"
 	l2ethclient "github.com/mantlenetworkio/mantle/l2geth/ethclient"
-	"github.com/mantlenetworkio/mantle/l2geth/rollup"
```

Comment:
Please adjust the grouping of imports according to the code style guidelines.

Reply:
done